pkg/chartrenderer/default.go

```diff
@@ -160,31 +160,48 @@ func (c *RenderedChart) Manifest() []byte {
 }
 
 // Files returns all rendered manifests mapping their names to their content.
-func (c *RenderedChart) Files() map[string]string {
-	var files = make(map[string]string)
+func (c *RenderedChart) Files() map[string]map[string]string {
```

Comment:
Can you elaborate more on the motivation for this commit because I don't get it from the commit message.
Is the the issue that in the previous commit `manifest.SplitManifests(files)` invocation was dropped from the `renderResources` func? If yes, can you elaborate why it was dropped? I see that there is the same func in helm3 but in different pkg - `releaseutil.SplitManifests`.

Reply:
It would be useful if you have provided links to issue or funcs. Anyways, I will try to check.

Reply:
helm@v2.17 seems to have exactly the same `SplitManifests` func as helm@v3.10:
helm@v2.17: https://github.com/helm/helm/blob/release-2.17/pkg/releaseutil/manifest.go#L37-L59
helm@v3.10: https://github.com/helm/helm/blob/release-3.10/pkg/releaseutil/manifest.go#L38-L60

Am I missing something?

Reply:
We are using [this](https://github.com/helm/helm/blob/a690bad98af45b015bd3da1a41f6218b1a451dbe/pkg/manifest/splitter.go#L32-L46) `SplitManifests` function, which doesn't split file properly.

Reply:
Ok thanks, I see that `pkg/manifest.SplitManifests` is removed in helm@v3. As far as I see, `releaseutil.SortManifests` calls internally `releaseutil.SplitManifests`. For a manifest file with multiple resources inside, the result from `releaseutil.SortManifests` contains `Manifest`s for every resource with the same name (the file name) and with the resource content. I think this is the change that requires adaptation of the `#Files()`, `#FileContent()` funcs.

PS: I added a release note of type `breaking dependency` for the changed signature of the `#Files()` func.

Reply:
Yes, exactly. Thanks 😃.